src/ostorlab/serve_app/common.py

```diff
@@ -27,6 +29,57 @@ class RiskRatingEnum(graphene.Enum):
     INFO = "Info"
 
 
+class Bytes(scalars.Scalar):
```

Comment:
Where do we need that? This is for ubjson.

Reply:
it is used for the value of AgentArgumentInputType

Reply:
This means we need `ubjson` on the OXO API.

Reply:
@3asm added, please check